README.md

```diff
@@ -22,8 +22,18 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
     end
     ```
 
+## Configuration with env vars
+Insert this in your Phoenix configuration files. Eg:  `config.exs`
```

Comment:
Will this config work in non-Phoenix setups? If so, maybe this should read: "Insert this in your Mix or Phoenix configuration files"

Reply:
I'd prefer to leave this as is unless anyone can confirm that the config actually works in a non-Phoenix setup.

Reply:
Confirmed as working in a regular Mix project set up via ```mix new```.

I might also recommend showing the alternate configuration of placing the data directly in the config file:
```elixir
config :powerdnsex, url: "localhost:8081",
                    token: "sometoken"
```
This is less complex and would work fine -- I could see possibly using that style and placing the config file via server configuration management software, such as Salt/Chef/Puppet/Ansible.

Reply:
Ok, I will leave both ways to configure PowerDNSex in README, so people can choose what is best for them.

I particularly prefer using env vars, which follows 12 Factor App, and makes your life easier when you have to change your environment.

Reply:
Using Salt (and I'm guessing most others of its kind), it's trivial to manage the config file differently based on environment. I can't even imagine managing servers 'manually' these days :)

Reply:
Yeah, I think so. Anyway, having two ways to configure should be good.